### PR TITLE
Improve Gerrit icon

### PR DESCRIPTION
**Pull request by felipecrs**
SVG was taken from https://en.m.wikipedia.org/wiki/File:Gerrit_icon.svg, and pretty-formatted with the XML extension for VS Code.

Then, I used the following tool to generate the PNG in 256x256 which is the size [recommended by VS Code](https://code.visualstudio.com/api/references/extension-manifest).

https://vincerubinetti.github.io/svg-to-png/

### CODE REVIEW

**Comment by SanderRonde** [comment]
Thanks for the PR! The PNG size upgrade is a good change, let's keep that one. But the svg change will actually make the icon look really weird in VSCode (see image below). VSCode turns the colored in SVG into a greyscale one with only filled-in or transparant. The red and green then also turn into grey turning it into a square blob. If you can remove that part of the change from this PR it's all good!
image

**Comment by felipecrs** [comment]
You are totally right. I'll see what I can do, otherwise I'll restore the original SVG.

**Comment by felipecrs** [comment]
I restored the original icon, but pretty-printed it with the [SVG extension](https://marketplace.visualstudio.com/items?itemName=jock.svg).

I noticed a very small issue with it:

image

But I could not fix it, I don't know anything about SVGs lol.

File: vscode--gerrit-1.2.34.zip
Comment (SanderRonde) [inline vscode--gerrit-1.2.34.zip:1]: Oh wow did I commit this one. thanks for removing it

File: yarn.lock
Comment (SanderRonde) [inline yarn.lock:1]: This one was actually intentional, it makes the git diff a bit more readable (see [bun docs](https://bun.sh/docs/install/lockfile))

**Review by SanderRonde — CHANGES_REQUESTED** [review]

**Comment by SanderRonde** [comment]
Ah the little bit of space between the svg is fine, you won't see it anyways since it's so tiny. So if you restore the `yarn.lock` file all good

File: yarn.lock
Comment (felipecrs) [reply yarn.lock:1]: Oh, got it. I restored it and forcefully set vscode to use bun, otherwise:

image

**Comment by felipecrs** [comment]
> Ah the little bit of space between the svg is fine, you won't see it anyways since it's so tiny. So if you restore the `yarn.lock` file all good

I actually fixed it in https://github.com/SanderRonde/VSCode-Gerrit/pull/75/commits/acf6609737da3af96266f12641777072d9f55709.

File: yarn.lock
Comment (felipecrs) [reply yarn.lock:1]: PS: I just noticed you did the same thing too on main branch. Restored it now.

**Review by SanderRonde — APPROVED** [review]

**Comment by SanderRonde** [comment]
Awesome, thanks for the PR! Will merge it as soon as you fix the merge conflicts :)

**Comment by felipecrs** [comment]
> Awesome, thanks for the PR! Will merge it as soon as you fix the merge conflicts :)

Hm... I had fixed already, I believe.

image

**Comment by SanderRonde** [comment]
image

Oh weird github is showing me something else. I'll try and fix it

**Comment by SanderRonde** [comment]
Ah it was the rebasing, a merge commit was fine. Thanks again for the PR!

**Pull request merged**